src/bin/julialauncher.rs

```diff
@@ -300,6 +302,181 @@ fn get_override_channel(
     }
 }
 
+fn get_project(args: &Vec<String>) -> Option<PathBuf> {
+    let mut project_arg: Option<String> = None;
+    for (_, arg) in args.iter().enumerate() {
+        if arg.starts_with("--project=") {
+            project_arg = Some(arg["--project=".len()..].to_string());
+            // Note: You'd think this might work, but it's not actually supported.
+            // } else if arg == "--project" && i + 1 < args.len() {
+            //     project_arg = Some(args[i + 1].clone());
+        } else if arg == "--" {
+            break;
+        }
+    }
```

Comment:
This loop still needs to be modified so that it doesn't pick up things like `julia main.jl --project=@.`.

Reply:
Ah yes, thanks. Done.